recipes/capstone/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.2)
```

Comment:
Our defacto has been `3.1` but I noticed you've been using `3.2`  recently... just a note for your template =)

Reply:
so, should it be 3.1 or 3.2? for what reason? I don't use any fancy CMake features.

Reply:
KB-H048 + KB-H049 explains it 

https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h048-cmake-version-required

Reply:
let's use 3.1 then